### PR TITLE
W-10862051 unique id content

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/custom-ingress-configuration.adoc [hunk 1]
```diff
@@ -193,8 +193,21 @@ labels:
   environment: 
   environment-id
 ----
+|`unique-id` | `host` a|
```
Comment (hegderohit89) [inline modules/ROOT/pages/custom-ingress-configuration.adoc]: its supported in all `path`, `host`, `annotation` and `labels`

File: modules/ROOT/pages/custom-ingress-configuration.adoc [hunk 2]
```diff
@@ -193,8 +193,21 @@ labels:
...
 ----
+|`unique-id` | `host` a| 
+
+[source,yaml]
+----
+host: app-name-unique-id.com
+----
 |===
 
+Use the `unique-id` placeholder to create a unique application URL, for example, if you want to deploy two applications with same name from different business groups.
```
Comment (hegderohit89) [inline modules/ROOT/pages/custom-ingress-configuration.adoc]: @kmendiratta-crm Do you foresee questions like:
1. How is this uniqueId created?
2. What kind of key it is ? (i mean its a shard string) etc

File: modules/ROOT/pages/custom-ingress-configuration.adoc (hunk 1 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/custom-ingress-configuration.adoc]: got it

File: modules/ROOT/pages/custom-ingress-configuration.adoc (hunk 2 above)
Comment (hegderohit89) [reply modules/ROOT/pages/custom-ingress-configuration.adoc]: @sarjeet2013

File: modules/ROOT/pages/custom-ingress-configuration.adoc [hunk 3]
```diff
@@ -193,8 +193,21 @@ labels:
...
+
+[NOTE]
+--
+The `unique-id` placeholder is available in Runtime Fabric version 1.12.(???) and later.
```
Comment (hegderohit89) [inline modules/ROOT/pages/custom-ingress-configuration.adoc]: @IsaacEldridge 
```suggestion
The `unique-id` placeholder is available in Runtime Fabric version 1.12.28 and later.
```

**Review by rohit-verma7 — DISMISSED** [review]
LGTM

**Review by emileemooreVT — DISMISSED** [review]

File: modules/ROOT/pages/custom-ingress-configuration.adoc (hunk 2 above)
Comment (hegderohit89) [reply modules/ROOT/pages/custom-ingress-configuration.adoc]: I have updated the Notes section with example rendered value

**Review by rohit-verma7 — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/custom-ingress-configuration.adoc [hunk 4]
```diff
@@ -193,8 +193,21 @@ labels:
...
 ----
+|`unique-id` | `path`, `annotation, labels` a| 
+
+[source,yaml]
+----
+host: app-name-unique-id.com
+----
 |===
 
+Use the `unique-id` placeholder to create a unique application URL. This is useful if you want to deploy two applications with same name from different business groups. For example, if you use `app-name-unique-id.com` in the `path` section of the ingress template, the resulting endpoint wil be similar to `app-name-7jkbic.com`.
```
Comment (hegderohit89) [inline modules/ROOT/pages/custom-ingress-configuration.adoc]:
```suggestion
Use the `unique-id` placeholder to create a unique application URL. This is useful if you want to deploy two applications with same name from different business groups. For example, if you use `app-name-unique-id.com` in the `path` section of the ingress template, the resulting endpoint will be similar to `app-name-7jkbic.com`.
```

**Review by sarjeet2013 — DISMISSED** [review]
lgtm

**Review by fermujica — APPROVED** [review]

**Pull request merged**